### PR TITLE
Add English recources for About screen

### LINKED ISSUE #142: Add English resource for string resource on About screen.
## Overview (Required)
- There is no English resource for string resource on About screen.

## Links
- https://github.com/DroidKaigi/conference-app-2019/blob/master/feature/about/src/main/res/values/strings.xml

### PR DESCRIPTION
**Pull request by kyoro**
## Issue
- close #142

## Overview (Required)
- Add strings.xml resources in English for About screen

## Links
- https://github.com/DroidKaigi/conference-app-2019/blob/master/feature/about/src/main/res/values/strings.xml

### CODE REVIEW

File: feature/about/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-    <string name="about_what_is_droidkaigi">What is DroidKaigi</string>
-    <string name="about_description_droidkaigi">DroidKaigiはエンジニアが主役のAndroidカンファレンスです。 \nAndroid技術情報の共有とコミュニケーションを目的に、2019年2月7日(木)、8日(金)の2日間開催します。</string>
-    <string name="about_access_to_place">会場アクセス</string>
-    <string name="about_check_map">Mapを開く</string>
-    <string name="about_staff_list">スタッフリスト</string>
-    <string name="about_check">見てみる</string>
-    <string name="about_privacy_policy">プライバシーポリシー</string>
-    <string name="about_license">ライセンス</string>
-    <string name="about_app_version">アプリバージョン</string>
+    <string name="about_what_is_droidkaigi">About DroidKaigi</string>
```
Comment (jmatsu) [inline feature/about/src/main/res/values/strings.xml]: Could you please revert this? This should be the same to the web's sentence.

File: feature/about/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
+    <string name="about_description_droidkaigi">DroidKaigi is a conference tailored for Android developers. It\'s schedule to take place on the 7th and 8th of February 2019.</string>
```
Comment (jmatsu) [inline feature/about/src/main/res/values/strings.xml]: We found a typo in our web page 😂 Could you please fix `It\'s schedule` to `It\'s scheduled`?

File: feature/about/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
+    <string name="about_access_to_place">Directions</string>
```
Comment (jmatsu) [inline feature/about/src/main/res/values/strings.xml]: We are using `Access` in our web page.

File: feature/about/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -1,13 +1,13 @@
...
+    <string name="about_check_map">Open with Maps</string>
+    <string name="about_staff_list">Operation Team</string>
```
Comment (jmatsu) [inline feature/about/src/main/res/values/strings.xml]: We haven't used *Operation Team* as our members' name. Just `Staffs` is better.

File: feature/about/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -1,13 +1,13 @@
...
+    <string name="about_check">See Details</string>
+    <string name="about_privacy_policy">Privacy Policy</string>
+    <string name="about_license">Open Sources Licenses</string>
```
Comment (jmatsu) [inline feature/about/src/main/res/values/strings.xml]: `Sources` -> `Source`

File: feature/about/src/main/res/values/strings.xml (hunk 1 above)
Comment (kyoro) [reply feature/about/src/main/res/values/strings.xml]: In my opinion, using interrogative sentence is not fit in this context. However, I also understood your opinion about wanting to use the same word in the official website.
I revert it.

File: feature/about/src/main/res/values/strings.xml (hunk 2 above)
Comment (kyoro) [reply feature/about/src/main/res/values/strings.xml]: This is just copied and pasted!

File: feature/about/src/main/res/values/strings.xml (hunk 3 above)
Comment (kyoro) [reply feature/about/src/main/res/values/strings.xml]: I think `Directions` is better in English, what do you think?

File: feature/about/src/main/res/values/strings.xml (hunk 5 above)
Comment (kyoro) [reply feature/about/src/main/res/values/strings.xml]: img
I referenced Google I/O app.

File: feature/about/src/main/res/values/strings.xml (hunk 4 above)
Comment (kyoro) [reply feature/about/src/main/res/values/strings.xml]: I think `Staffs` is meaning multiple of ingredients or original materials.
If using for human, just `stuff` is better.
However, stuff is not the best word for the user community-based conference. I'm feeling employee-ish for this word....

Comment (KeithYokoma) [reply feature/about/src/main/res/values/strings.xml]: No problem! These two are too confusing 😜

Comment (kyoro) [reply feature/about/src/main/res/values/strings.xml]: I'm confused, I don't see 'Staffs' in my life. I saw wrong. Sorry!

Comment (kyoro) [reply feature/about/src/main/res/values/strings.xml]: Anyway, `Committee Member` looks good for me 👍

File: feature/about/src/main/res/values/strings.xml (hunk 5 above)
Comment (kyoro) [reply feature/about/src/main/res/values/strings.xml]: I'm feeling wrong wording for `sources`... I fixed it.

File: feature/about/src/main/res/values/strings.xml (hunk 3 above)
Comment (panpanini) [reply feature/about/src/main/res/values/strings.xml]: I agree with @kyoro here - even if its different from the website, I think it makes sense in this context

File: feature/about/src/main/res/values/strings.xml [hunk 6]
```diff
@@ -1,13 +1,13 @@
...
-    <string name="about_staff_list">スタッフリスト</string>
-    <string name="about_check">見てみる</string>
-    <string name="about_privacy_policy">プライバシーポリシー</string>
-    <string name="about_license">ライセンス</string>
-    <string name="about_app_version">アプリバージョン</string>
+    <string name="about_what_is_droidkaigi">What is DroidKaigi?</string>
+    <string name="about_description_droidkaigi">DroidKaigi is a conference tailored for Android developers. It\'s scheduled to take place on the 7th and 8th of February 2019.</string>
+    <string name="about_access_to_place">Directions</string>
+    <string name="about_check_map">Open with Maps</string>
+    <string name="about_staff_list">Committee Members</string>
```
Comment (panpanini) [inline feature/about/src/main/res/values/strings.xml]: maybe `DroidKaigi Members` is better? (`Committee Members` sounds very formal, so I'm not sure if I would click that button if I saw it)

File: feature/about/src/main/res/values/strings.xml (hunk 3 above)
Comment (jmatsu) [reply feature/about/src/main/res/values/strings.xml]: Sure. 👀 
@kyoro Sorry for bothering you due to my review. We will go with *Directions*.

File: feature/about/src/main/res/values/strings.xml (hunk 6 above)
Comment (kyoro) [reply feature/about/src/main/res/values/strings.xml]: @panpanini Thanks for your following! 👍 
I think `DroidKaigi Members` should be works, but we(DroidKaigi) looks already using `committee` on the official website...?

img

Comment (jmatsu) [reply feature/about/src/main/res/values/strings.xml]: 🤔 Okay, let's go with the word *Committee Members*! We do not need to rethink this word until other English speakers feel weird.

**Pull request merged**